bootstrapper/internal/helm/helm.go

```diff
@@ -188,6 +189,16 @@ func (h *Client) installCiliumGCP(ctx context.Context, kubectl k8sapi.Client, re
 // The function will wait 30 seconds before retrying a failed installation attempt.
 // After 5 minutes the retrier will be canceld and the function returns with an error.
 func (h *Client) install(ctx context.Context, chartRaw []byte, values map[string]any) error {
+	retriable := func(err error) bool {
+		if errors.Is(err, wait.ErrWaitTimeout) {
+			return true
+		} else if strings.Contains(err.Error(), "connection refused") {
+			return true
+		} else {
+			return false
+		}
```

Comment:
```suggestion
		switch {
		case errors.Is(err, wait.ErrWaitTimeout):
			return true
		case strings.Contains(err.Error(), "connection refused"):
			return true
		default:
			return false
		}
```
I like the switch syntax a lot :D. But do as you like.

Reply:
I'd even shorten this to
```go
	return errors.Is(err, wait.ErrWaitTimeout) ||
		strings.Contains(err.Error(), "connection refused")
```
but opinions on readability may differ